.github/workflows/bump-version.yml

```diff
@@ -12,26 +12,77 @@ on:
           - patch
           - minor
           - major
+  issue_comment:
+    types: [created]
 
 jobs:
   bump-version:
     name: 🔢 Bump Version
     runs-on: ubuntu-latest
+    if: >
+      github.event_name == 'workflow_dispatch' ||
+      (github.event_name == 'issue_comment' && 
+       github.event.issue.pull_request &&
+       startsWith(github.event.comment.body, '/bump-version'))
     steps:
+      - name: 🔍 Parse Comment Command
```

Comment:
This seems like a really convoluted way to accomplish this; is there a simpler way?